profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php

```diff
@@ -541,4 +614,31 @@ public static function prepareReferenceNumber($demande_id) {
       . '_' . $demande_id['produit'];
   }
 
+  /**
+   * Returns 'Demande ID' array based on given job reference.
```

Comment:
semantically names demande_id that is an array can make confusion, would it be better to rename it to demande or demande_array or something else?